README.md

```diff
@@ -1,4 +1,7 @@
-# invert-api
+<p align="center">
+  <img src="https://raw.githubusercontent.com/Mentors4EDU/Images/master/invertapilogo.png">
```

Comment:
I kind of liked the pyramid design, it hints at being an Egyptian and the top-down inverted power structure that OSS brings.